FluentTerminal.Client/src/style.css

```diff
@@ -1,15 +1,24 @@
 * {
     margin: 0;
+    -ms-overflow-style: -ms-autohiding-scrollbar;
```

Comment:
Can you try to make this configurable again? https://github.com/felixse/FluentTerminal/blob/8131140cb0f358826e46ebcc69cb4cec2c971598/FluentTerminal.Client/src/index.js#L147